package.json

```diff
@@ -3,24 +3,35 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/babel-plugin": "11.0.0",
+    "@emotion/react": "11.1.1",
+    "@emotion/styled": "11.0.0",
+    "@material-ui/core": "4.11.2",
+    "@material-ui/styles": "4.11.2",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^10.4.9",
     "@testing-library/user-event": "^12.1.3",
+    "axios": "0.21.0",
+    "dotenv": "^8.2.0",
+    "jest": "24.9.0",
```

Comment:
Is there a reason to include this version of the jest library? by default the `create-react-app` includes this package